lib/internal/test_runner/harness.js

```diff
@@ -76,6 +76,7 @@ function createTestTree(rootTestOptions, globalOptions) {
 
       buildPhaseDeferred.resolve();
     },
+    testsProcesses: new SafeMap(),
```

Comment:
Is this necessary? It seems to only be used in one file.

Reply:
Removed this in favour of @atlowChemi's suggested AbortController